VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,245 @@
+# Validators FAQ
+
+## General concepts
+
+### What is a validator?
+
+Validators are the pillars of the network. Their role is to run a full node and participate in consensus by broadcasting votes which contain cryptographic signatures signed by their private key. Validators relay themselves to commit new blocks in the blockchain and receive rewards in exchange for their work. They must also participate in governance by voting on proposals.
+
+### What is a full node?
+
+A full node is a program that fully validates transactions and blocks of a blockchain. It is distinct from a light node that only processes block headers and a small subset of transactions. Running a full node requires more resources than a light-node but is necessary in order to be a validator. In practice, running a full node only implies running a non-compromised and up-to-date version of the software with low network latency and without down time. 
+
+### What is a delegator?
+
+Delegators are ATOMs holders who  cannot ordo not want to run validator operations, which involve IT admin and secure data center, themselves. Through the Cosmos UI, a user can delegate to a validator and obtain a part of its reward in exchange.
+
+### How to become a validator?
+
+Any participant in the network can signal that they want to become a validator by sending a “self-designation” transaction, where they must fill out the following parameters:
+
+* Validator's name
+* Validator's website
+* Validator's description
+* Initial commission rate
+* Minimum self-bond amount
+* Initial self-bond amount
```

Comment:
These are values that the validator need to input. Aren't CommissionChangeToday and PubKey automatically set ? Why would the validator need to fill these values itself ?

Reply:
A validator does not need to set PubKey themselves in that transaction.

It needs to be obvious that some information, such as website or description can just be an empty string. In essence some of this information is optional. However initial commission rate isn't optional.

Reply:
> However initial commission rate isn't optional.

Who said it was ?

Reply:
Just an example that some things are optional and some aren't.

Reply:
Ok, will update to show what's optional and what's not

Reply:
@gamarin2 pubkey not set because can and should declare candidacy with a pubkey other than the one you are validating with - (aka don't use same account to hold liquid funds and validate) - you were correct about CommissionChangeToday removed from my comment

Reply:
@adrianbrink you must set your pubkey when you are declare candidacy - This is the core index for everything... I will think about how you could possible change your pubkey under unusual circumstances.. but that's post launch (opening issue now)

Reply:
addressed